README.md

```diff
@@ -138,8 +149,10 @@ You can change "default" to point to whichever JDBC connection you want to query
 You will need to define a connection for every database that will need to be access during the test run.
 For example, if you'd like the framework to create tables for you in Hive, you'll have to specify connection
 parameters for Hive.
-TODO: what if you've already created all your tables in hive, do you still need
-to provide connection parameters?
+
+Remark: Currently all database connections defined in the configuration are initialized eagerly even
```

Comment:
Remark: Every database connection defined in the configuration will be initialized during framework startup, even if no tests using that type of connection are scheduled to be executed.  This will be improved in the future.